README.md

```diff
@@ -1,2 +1,68 @@
 # dws-slurm-bb-plugin
 A Lua script for Slurm’s Burst Buffer plugin that maps DataWarp-style data movement directives to Workflows in Data Workflow Services.
+
+## Using the test environment as a playground
+
+The test environment may be used a playground to experiment with Slurm, DWS, and the dws-test-driver.
+
+Begin by setting up the test environment.  This will start KIND to create a k8s cluster where it will launch DWS and the dws-test-driver.  This will also run Slurm's `slurmctld` and `slurmd` containers.  The Slurm containers will not be running in the k8s cluster, but they are able to communicate with the DWS API.
+
+Note: this is a minimalist Slurm environment and it does not support all Slurm functionality.
+
+```console
+$ make -C testsuite/integration setup
+```
+
+Enter the `slurmctld` container to use Slurm's commands.  Jobs must be launched by the `slurm` user.
+
+```console
+$ docker exec -it slurmctld bash
+
+[root@slurmctld jobs]# su slurm
+bash-4.4$ cd /jobs
+```
+
+The `/jobs` directory is mounted into the container from your workarea.  You can find it in your workarea at `testsuite/integration/slurm/jobs`.  This directory contains the job script used for the integration test and also a sample job script.  Any output files from job scripts will also be stored in this directory.  Slurm commands such as `sbatch`, `scontrol`, and `scancel` may be used from this location in the container if run as the `slurm` user.
```

Comment:
```suggestion
The `/jobs` directory is mounted into the container from your workarea.  You can find it in your workarea at `testsuite/integration/slurm/jobs`.  This directory contains a sample job script.  Any output files from job scripts will also be stored in this directory.  Slurm commands such as `sbatch`, `scontrol`, and `scancel` may be used from this location in the container if run as the `slurm` user.
```

Reply:
The integration test script is not used during integration testing. All scripts used during integration testing are included in the feature files. Just a minor clarification.